docs/data_management/lts/iam_and_policies.md

```diff
@@ -1,4 +1,84 @@
+    - Access Key: The public identifier used to access the allocation, similar to a username.
+    - Secret Key: The private password-like credential that must be kept confidential.
+- **Stewards**: Stewards are individuals responsible for managing an allocation. Stewards need a full access key pair to perform tasks like creating, deleting, and maintaining buckets. Each steward must maintain separate key pairs for their allocations and any lab/Core allocations they manage.
+- **Owner**: owners responsible for overseeing the management of allocated storage spaces, ensuring compliance with data management policies, and designating appropriate stewards, if needed, to assist in the management of allocation. Owners can include:
+    - **Lab PIs**: Manage allocated storage spaces for their respective labs.
+    - **Core Director**: Manage allocated storage spaces for Core facilities.
+- **Bucket Policy**: A bucket policy is a [JSON formatted file](https://docs.fileformat.com/web/json/) that you can use to grant access permissions to your LTS bucket and the objects in it. Please refer to [sharing buckets and bucket policy structure](../lts/iam_and_policies.md#sharing-buckets) for more details.
+- **Identity and Access Management (IAM)**: IAM is a framework of policies, processes, and technologies used to manage digital identities and control access to resources.
+
+## Shared LTS Allocations
+
+Shared LTS allocations are storage spaces designed for collaborative use by groups and are available to Lab PIs and Core Directors. If a PI is also a Core Director, they are eligible for independent storage allocations for each organization: one for the Lab and one for the Core. The shared LTS allocation provides 75 TB of storage. These allocations can be named according to the preference of the PI or Core Director. For recommended naming guidelines, refer to our [Naming Shared Storage](../../data_management/storage.md#how-do-i-request-shared-storage) documentation.
+
+{{ read_csv('data_management/res/shared_allocation_functional_roles.csv', keep_default_na=False) }}
+
+### How do I grant other users access to my shared LTS Allocation?
+
+By default, only the allocated owner, and stewards if designated, can manage and access a shared LTS allocation using their specific key pairs. Keys from other allocations, such as those for individual LTS allocations, will not grant access to shared lab or core allocations. If you manage both lab and core allocations, ensure you use the corresponding keys, as keys for one cannot access the other.
+
+To grant other users access to your shared allocation:
+
+- The Owners, or a steward if one has been designated, can set permissions using a [bucket policy](../lts/iam_and_policies.md#sharing-buckets).
+- Access can be granted to any user with system access, including those with individual LTS allocations and an active Cheaha account.
```

Comment:
This sentence feels ambiguous, but I'm not certain of the intent.

Maybe "...granted to any user with an individual LTS allocation." If this sounds right, I think it would help to be brief and specific here.

Reply:
I don't think we need to mention the Cheaha account. When they go to request an LTS allocation, we can check then. If we want to document the need for a Cheaha account, let's be sure we're doing that wherever that section is, rather than here.

Reply:
That is correct. The intent was to outline the criteria that qualify individuals to be granted access to the shared LTS allocation. I have now shortened it to focus on the individual LTS, as the Cheaha account is handled elsewhere, as you mentioned. Thanks!